docs/source/reference/numpysupported.rst

```diff
@@ -251,6 +251,7 @@ The following methods of Numpy arrays are supported:
 
 * :meth:`~numpy.ndarray.transpose`
 * :meth:`~numpy.ndarray.view` (only the 1-argument form)
+* :meth:`~numpy.ndarray.__contains__` (only the 1-argument form ``key``)
```

Comment:
What does "(only the 1-argument form ``key``)" mean? Should this just be

```suggestion
* :meth:`~numpy.ndarray.__contains__`
```

?

Reply:
Here, I was trying to frame `__contains__` as a method that takes a single argument `key`, for use as `key in some_array`. It does sound more understandable without  "(only the 1-argument form `key`)" though.